README.md

```diff
@@ -1,18 +1,49 @@
 # Azure Exporter for Scrapy
+[Scrapy feed export storage backend](https://doc.scrapy.org/en/latest/topics/feed-exports.html#storage-backends) for [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/).
 
-## Requirements:
-- Python 3.8
-- Pipenv
```

Comment:
The minimum Python version is worth mentioning in the README. Maybe keep the Requirements section and only remove Pipenv?